src/enrollments/enrollments.service.ts

```diff
@@ -65,4 +66,16 @@ export class EnrollmentsService {
 
     await this.em.removeAndFlush(enrollment);
   }
+
+  async getStudentsForClassroom(classroomId: number): Promise<IEnrollment[]> {
+    const classroom = await this.classroomRepository.findOneOrFail({ id: classroomId });
+
+    const enrollments = await this.enrollmentRepository.find(
+      { classroom },
+      {
+        populate: ["student", "student.user"],
+      },
+    );
+    return enrollments;
```

Comment:
You might not need the extra variable. Instead, just use return and it should resolve the promise

Reply:
Bhaiya, I do that because there is a lint warning for return and await in the same line.